plasmapy_nei/nei/nei.py

```diff
@@ -1,7 +1,1359 @@
+        The electron temperature, which may be a constant, an array of
+        temperatures corresponding to the times in `time_input`, or a
+        function that yields the temperature as a function of time.
+
+    n: astropy.units.Quantity or callable
+        The number density multiplicative factor.  The number density of
+        each element will be ``n`` times the abundance given in
+        ``abundances``.  For example, if ``abundance['H'] = 1``, then this
+        will correspond to the number density of hydrogen (including
+        neutral hydrogen and protons).  This factor may be a constant,
+        an array of number densities over time, or a function that
+        yields a number density as a function of time.
+
+    time_input: astropy.units.Quantity, optional
+        An array containing the times associated with ``n`` and ``T_e`` in
+        units of time.
+
+    time_start: astropy.units.Quantity, optional
+        The start time for the simulation.  If density and/or
+        temperature are given by arrays, then this argument must be
+        greater than ``time_input[0]``.  If this argument is not supplied,
+        then ``time_start`` defaults to ``time_input[0]`` (if given) and
+        zero seconds otherwise.
+
+    time_max: astropy.units.Quantity
+        The maximum time for the simulation.  If density and/or
+        temperature are given by arrays, then this argument must be less
+        than ``time_input[-1]``.
+
+    max_steps: `int`
+        The maximum number of time steps to be taken during a
+        simulation.
+
+    dt: astropy.units.Quantity
+        The time step.  If ``adapt_dt`` is `False`, then ``dt`` is the
+        time step for the whole simulation.
+
+    dt_max: astropy.units.Quantity
+        The maximum time step to be used with an adaptive time step.
+
+    dt_min: astropy.units.Quantity
+        The minimum time step to be used with an adaptive time step.
+
+    adapt_dt: `bool`
+        If `True`, change the time step based on the characteristic
+        ionization and recombination time scales and change in
+        temperature.  Not yet implemented.
+
+    safety_factor: `float` or `int`
+        A multiplicative factor to multiply by the time step when
+        ``adapt_dt`` is `True`.  Lower values improve accuracy, whereas
+        higher values reduce computational time.  Not yet implemented.
+
+    tol: float
+        The absolute tolerance to be used in comparing ionic fractions.
+
+    verbose: bool, optional
+        A flag stating whether or not to print out information for every
+        time step. Setting ``verbose`` to `True` is useful for testing.
+        Defaults to `False`.
+
+    abundances: dict
+
+    Examples
+    --------
+
+    >>> import numpy as np
+    >>> import astropy.units as u
+
+    >>> inputs = {'H': [0.9, 0.1], 'He': [0.9, 0.099, 0.001]}
+    >>> abund = {'H': 1, 'He': 0.085}
+    >>> n = np.array([1e9, 1e8]) * u.cm ** -3
```

Comment:
Could have been a
```suggestion
    >>> n = u.Quantity([1e9, 1e8], u.cm**-3)
```

just saying, it's a slightly faster alternative I don't see people using.